R/check_description.R

```diff
@@ -250,12 +251,25 @@ Please send a pull request if you need support for this license.",
   }
 
   # check if LICENSE.md matches the official version
-  current <- readLines(file.path(x$get_path, "LICENSE.md"))
+  current <- switch(
+    this_desc$get_field("License"),
+    "GPL-3" = readLines(file.path(x$get_path, "LICENSE.md")),
+    "MIT" = readLines(file.path(x$get_path, "LICENSE.md"))[-(1:4)]
```

Comment:
We should be thorough and check the full license.

- check the presence of the header
- check for a correct copyright notice
    - does it contain the year
    - does the copyrightholder matches with the one listed in the `DESCRIPTION`.

Reply:
See [3c0b033](https://github.com/inbo/checklist/pull/92/commits/3c0b033f08e7d17729b551d9c0808edb50d2b4da)